package-lock.json

```diff
@@ -1,4419 +1,8 @@
 {
   "name": "@mapbox/node-pre-gyp",
   "version": "1.0.9",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
looks like you need to update your local version of npm to avoid downgrading the lockfile